docs/guides/api-rest/express-server.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Express server
+description: How to deploy an Express server to AWS using AWS Amplify
+---
+
+In this guide you'll learn how to deploy an [Express](https://expressjs.com/) web server complete with routing.
+
+## Getting started
```

Comment:
remove this if it's the only header. This would get rid of the right hand nav to make it easier to read.